python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5688,3 +5689,174 @@ def test_dataframe_assign_scalar_with_scalar_cols(col_data, assign_val):
     gdf["b"] = assign_val
 
     assert_eq(pdf, gdf)
+
+
+# flake8: noqa : W291
+def test_dataframe_info():
```

Comment:
Can we split these into separate tests? Can we also add a test for when we have null values and ask for `null_counts=True`?

Reply:
Sure, made the changes.